sync/wizard/sync_make_module.py

```diff
@@ -80,6 +81,8 @@ def act_makefile(self):
             )
         )
         root = etree.Element("odoo")
+        root_data = etree.SubElement(root, "data")
+        root_data.set("noupdate", str(int(self.data_noupdate)))
```

Comment:
I'd prefer to skip adding if `self.data_noupdate` is False
```suggestion
        if self.data_noupdate:
            root_data.set("noupdate", str(int(self.data_noupdate)))
```